kubevirtci/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to kubevirt/kubevirtci
```

Comment:
Do we not have the option to consume just the parts of `kubevirtci` that we're interested in?
We don't need all this stuff

Reply:
I think we should rather have one subtree then a bunch of them.